css/properties/rotate.json

```diff
@@ -0,0 +1,116 @@
+{
+  "css": {
+    "properties": {
+      "translate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/translate",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "60",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "layout.css.individual-transform.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "60",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "layout.css.individual-transform.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "x_y_z": {
+          "__compat": {
+            "description": "Support for x, y, or z axis name plus angle value",
```

Comment:
Avoid support (it is implicit): "x, y, or z axis name plus angle value" or "Axis name (x, y, or z) plus angle value"